api/src/main/java/org/apache/cloudstack/api/command/admin/systemvm/MigrateSystemVMCmd.java

```diff
@@ -91,6 +97,10 @@ public Long getStorageId() {
         return storageId;
     }
 
+    public Boolean isAutoSelect() {
+        return autoSelect != null ? autoSelect : true;
```

Comment:
Minor suggestion: to use `BooleanUtils.toBoolean(autoSelect)` here

Reply:
@nvazquez done. thanks for suggestion.

use `BooleanUtils.isNotFalse`
https://commons.apache.org/proper/commons-lang/javadocs/api-release/org/apache/commons/lang3/BooleanUtils.html#isNotFalse-java.lang.Boolean-